dynawo/sources/Models/Modelica/Dynawo/Examples/GridForming/GridForming.mo

```diff
@@ -0,0 +1,236 @@
+    Line(points = {{47, 135}, {40, 135}, {40, 155}, {100, 155}, {100, 151}, {100, 151}}, color = {0, 0, 127}));
+  connect(Conv500.iqConvPu, dVOC.iqConvPu) annotation(
+    Line(points = {{47, 131}, {39, 131}, {39, 156}, {104, 156}, {104, 151}, {104, 151}}, color = {0, 0, 127}));
+  connect(Conv500.iqPccPu, dVOC.iqPccPu) annotation(
+    Line(points = {{47, 126}, {38, 126}, {38, 157}, {109, 157}, {109, 151}, {109, 151}}, color = {0, 0, 127}));
+  connect(Conv500.uqFilterPu, dVOC.uqFilterPu) annotation(
+    Line(points = {{47, 122}, {37, 122}, {37, 158}, {114, 158}, {114, 151}, {114, 151}}, color = {0, 0, 127}));
+  connect(Line23Bis1.terminal2, Fault.terminal) annotation(
+    Line(points = {{-20, 74}, {0, 74}, {0, 44}}, color = {0, 0, 255}));
+  connect(Conv1000.udFilterPu, Matching.udFilterPu) annotation(
+    Line(points = {{-46, -13.5}, {-42, -13.5}, {-42, -17}, {-89, -17}, {-89, -16}}, color = {0, 0, 127}));
+  connect(Conv1000.idPccPu, Matching.idPccPu) annotation(
+    Line(points = {{-46, -9}, {-41, -9}, {-41, -18}, {-93, -18}, {-93, -16}}, color = {0, 0, 127}));
+  connect(Conv1000.idConvPu, Matching.idConvPu) annotation(
+    Line(points = {{-46, -4.5}, {-40, -4.5}, {-40, -19}, {-97.5, -19}, {-97.5, -16}}, color = {0, 0, 127}));
+  connect(Conv1000.UdcSourcePu, Matching.UdcSourcePu) annotation(
+    Line(points = {{-46, 0}, {-39, 0}, {-39, -20}, {-102, -20}, {-102, -16}}, color = {0, 0, 127}));
+  connect(Conv1000.iqConvPu, Matching.iqConvPu) annotation(
+    Line(points = {{-46, 4.5}, {-38, 4.5}, {-38, -21}, {-107, -21}, {-107, -16}}, color = {0, 0, 127}));
+  connect(Conv1000.iqPccPu, Matching.iqPccPu) annotation(
+    Line(points = {{-46, 9}, {-37, 9}, {-37, -22}, {-111, -22}, {-111, -16}}, color = {0, 0, 127}));
+  connect(Conv1000.uqFilterPu, Matching.uqFilterPu) annotation(
+    Line(points = {{-46, 13.5}, {-36, 13.5}, {-36, -23}, {-115.5, -23}, {-115.5, -16}}, color = {0, 0, 127}));
+  annotation(
+    experiment(StartTime = 0, StopTime = 3, Tolerance = 0.000001),
+    __OpenModelica_simulationFlags(initialStepSize = "0.001", lv = "LOG_STATS", nls="kinsol",  s = "ida", nlsLS = "klu",  maxIntegrationOrder = "2", maxStepSize = "10", outputFormat = "mat"),
+    Documentation(info = "<html><head></head><body><span style=\"font-size: 12px;\">
+    This test case consists in three different grid-forming converters (one with a droop control, one with a dispatchable virtual oscillator control and one with a matching control) connected to a load. At t = 0.5s the line connecting the 250MW and the 1000MW converters is opened. At t = 1.5s, a short-circuit occurs in the middle of one of the lines connecting the 250MW and the 500MW converters. It is cleared after 150ms. This test case and the grid-forming converters controls come from the Horizon 2020 European project MIGRATE, and more precisely from its Deliverables 3.2 and 3.3 that can be found on the project website : https://www.h2020-migrate.eu/downloads.html.
```

Comment:
- If possible it could be good to add a bit more space between converter and control so that we can read the variables names more clearly.

- It would be better to refer to the converter with their name or type of control rather than their power because it doesn't appear in the graphical view (or change MW to MVA because I believe you talk about their apparent power?).

- Do you think you can add some comments on the results or it is better to let people go read Migrate report? I think about comments about the power flow repartition after the event, the frequency value after the event (does it goes back to precedent value or not), the fact that currents are well limited during the fault... This is just some ideas, I don't know much about grid forming controls so I let you say what could be interesting! Otherwise it is already good. Thanks for the reference and the graphs!

Reply:
Thank you for you comments.
I tried to add some space and increase the variables size.
The type of control has been added in the documentation and I gave some explanations of the results as well as a mention of the deliverables.